zone/client_bot.cpp

```diff
@@ -15,10 +15,13 @@ void Client::SetBotOption(BotOwnerOption boo, bool flag) {
 	}
 }
 
-uint32 Client::GetBotCreationLimit(uint8 class_id)
-{
+uint32 Client::GetBotCreationLimit(uint8 class_id) {
 	uint32 bot_creation_limit = RuleI(Bots, CreationLimit);
 
+	if (Admin() >= RuleI(Bots, MinStatusToBypassCreateLimit)) {
+		return RuleI(Bots, StatusCreateLimit);
```

Comment:
Shouldn't this be more appropriately named like `MinStatusBypassCreateLimit` ? `StatusCreateLimit` seems ambiguous